clinicians/clinicians.go

```diff
@@ -14,7 +14,8 @@ var (
 	ErrNotFound  = fmt.Errorf("clinician %w", errors.NotFound)
 	ErrDuplicate = fmt.Errorf("%w: clinician is already a member of the clinic", errors.Duplicate)
 
-	ClinicAdmin = "CLINIC_ADMIN"
+	ClinicAdmin  = "CLINIC_ADMIN"
```

Comment:
Not a big deal, but I'm curious as to why these aren't constants?

Reply:
Good catch